api/spec/spec_helper.rb

```diff
@@ -62,7 +64,6 @@
   end
 
   config.include ActiveSupport::Testing::Assertions
```

Comment:
I think that without `ActiveJob::TestHelper` we can get rid of `ActiveSupport::Testing::Assertions` module too